### ISSUE TITLE
[Feature Request]: New module - Laravel JSON API

### ISSUE THREAD

**Issue by maicol07** [laravel-idea/plugin#501]
### Feature Description

Provide support for Laravel Json Api package: https://laraveljsonapi.io

**Comment by adelf** [comment]
Hi, Maicol.

What kind of support? generating schemas based on model fields? Or?

**Comment by maicol07** [comment]
Yes, basically the code generation, like the artisan commands (I'd like to use that primarly for packages that can't call `artisan` commands).

Example: [https://laraveljsonapi.io/docs/1.0/schemas/#defining-schemas](https://laraveljsonapi.io/docs/1.0/schemas/#defining-schemas) (command: `php artisan jsonapi:schema posts --server=v1`)

**Comment by maicol07** [comment]
@adelf this page shows the full list of generators included with the package: https://laraveljsonapi.io/docs/1.0/digging-deeper/artisan.html#generators

**Comment by maicol07** [comment]
@adelf any news about this one?

**Comment by adelf** [comment]
I won't implement it natively for now. But I'm working on ide.json changes and you can add these generations there byself.

**Comment by maicol07** [comment]
@adelf any news about this?

**Comment by adelf** [comment]
It's implemented but isn't documented yet. Working on it.
